tasks/control-plane-init.yml

```diff
@@ -61,14 +61,13 @@
     k8s_cluster_join_command: "{{ kubernetes_join_command_result.stdout | default('') }}"
   delegate_to: "{{ item }}"
   delegate_facts: true
-  with_items: "{{ groups['control_plane'] + groups['workers'] }}"
+  loop: "{{ groups['control_plane'] + groups['workers'] }}"
```

Comment:
Is there a way to avoid this solution? Ansible inventory groups hardcoding is a bad

Reply:
Could be be variables for groups, like [there](https://github.com/cloud-labs-infra/ansible-etcd-cluster/blob/ec990f914667bd94292901bd2f1a38d7c123f108/defaults/main.yml#LL8C1-L8C29)